web/src/lib/components/NostrChat.svelte

```diff
@@ -0,0 +1,297 @@
+            </div>
+            <div class="w-full">
+                <div class="flex items-center justify-between w-full">
+                    <div class="text-lg font-bold">{roomData.name || ''}</div>
+                </div>
+                <div>{roomData.about || ''}</div>
+            </div>
+        </div>
+    {/if}
+
+    <div class="ml-96 mr-96 flex flex-col py-2">
+        <div class="mt-4 mb-8">
+            <div tabindex="0" class="collapse collapse-plus border border-base-300 bg-base-100 rounded-box mb-4">
+                <input type="checkbox" />
+                <div class="collapse-title text-l font-medium bg-info">
+                    We use <b>Nostr</b> to power this chat. Click here to see more info
+                </div>
+                <div class="collapse-content bg-info">
+                    <p class="mb-4">If you prefer to participate in this chat using another Nostr client, you'll need one that support channels and subscribe to this channel ID: {nostrRoomId}</p>
+                </div>
+            </div>
+
+
+            <div class="flex flex-col">
+                <div class="form-control w-52">
+                    <label class="cursor-pointer label">
+                        <input id="use_browser_extension" type="checkbox" class="toggle toggle-primary mr-2" bind:checked={nostrPreferenceCheckboxChecked} on:change={e => {onChangeNostrPreferenceCheckbox(e)}} disabled />
+                        <span class="label-text">Use browser extension's identity</span>
+                    </label>
+                </div>
+                {#if nostrPreferenceCheckboxChecked}
+                    <small>You'll be asked to authorize signing messages when you want to write in the channel.</small>
+                {:else}
+                    <small>You'll be using your Plebeian Market generated Nostr identity. Install a Nostr browser extension
+                        (<a class="link" href="https://github.com/fiatjaf/nos2x" target="_blank">nos2x</a>,
+                        <a class="link" href="https://getalby.com/" target="_blank">Alby</a> or
+                        <a class="link" href="https://www.blockcore.net/wallet" target="_blank">Blockcore</a>) if you want to use your own Nostr identity:</small>
+                {/if}
+            </div>
+        </div>
+
+        <div class="flex border-base-300 bg-info-content-200 rounded-b-box rounded-tr-box
+                        min-h-[6rem] min-w-[18rem] max-w-4xl flex-wrap items-center justify-center
+                        gap-2 overflow-x-hidden border bg-cover bg-top p-4"
+             style="background-size: 5px 5px; background-image: radial-gradient(hsla(var(--bc)/.2) 0.5px,hsla(var(--b2)/1) 0.5px);"
+        >
+            <div class="w-full">
+                {#each sortedMessages as m}
+                    <div class="chat chat-start" class:mt-0={m.samePubKey === true} class:mt-5={m.samePubKey !== true} class:profileInfo={m.samePubKey !== true}>
```

Comment:
`{m.samePubKey}` / `{!m.samePubKey}` is not good? :)

Reply:
Oh no! I've used that at other places, but I should have had a bad day :)